packages/app/pages/course/[cid]/today.tsx

```diff
@@ -57,6 +58,7 @@ export default function Today(): ReactElement {
   }
   return (
     <div>
+      <WelcomeStudents />
```

Comment:
this is small, but it might be better to use a conditional render here so that we don't always render the modal in the DOM. if this messes up any ant design animations then no worries

`{firstTime && <WelcomeStudents />}`